tsup.config.js

```diff
@@ -0,0 +1,22 @@
+import autoprefixer from 'autoprefixer';
+import { defineConfig } from 'tsup';
+import postcssPlugin from '@chialab/esbuild-plugin-postcss';
+
+export default defineConfig(() => {
+  return {
+    clean: false,
+    dts: true,
+    format: ['esm', 'cjs'],
+    entry: ['./src/widget/inline.svelte', './src/widget/modal.svelte'],
+    watch: false,
+    declaration: true,
+    minify: false,
+    plugins: [
+      postcssPlugin({
+        plugins: [autoprefixer],
+      }),
+    ],
+    sourcemap: false,
```

Comment:
If we have `sourcemap` as false, are we preventing our ability to debug this code in the browser? I'd really like to preserve our ability to step through code in a meaningful way.

Reply:
I just havent had time to do any testing around sourcemaps in this version of hte story, so i left it as false for now.